src/app/constants/filesize.ts

```diff
@@ -1,2 +1,3 @@
-// 1 megabyte in bytes
-export const MB = 1048576
+export const bytesToKilobytes = (bytes: number) => (bytes / 1000).toFixed(2)
+export const bytesToMegabytes = (bytes: number) => (bytes / 1000000).toFixed(2)
+export const megabytesToBytes = (megabytes: number) => megabytes * 1000000
```

Comment:
Should we only have constants instead of a function? The file referencing the constant can do the appropriate calculations.

```ts
export const KB = 1000
export const MB = 1000 * KB
```